microsoft-edge/progressive-web-apps-chromium/get-started.md

```diff
@@ -0,0 +1,364 @@
+---
+description: This guide gives you an overview of PWA basics and tools for building progressive web apps on Windows.
+title: Get started with Progressive Web Apps
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 05/13/2020
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: progressive web apps, PWA, Edge, Windows, PWABuilder, web manifest, service worker, push
+---
+
+# Get started with Progressive Web Apps  
+
+Progressive Web Apps \(PWAs\) are web apps that are [progressively enhanced][WikiProgressiveEnhancement] with app-like features, such as installation, offline support, and push notifications.  You may also packaged your PWA for app stores, including the Microsoft Store, Google Play, Mac App Store and more.  
```

Comment:
I think we can keep Microsoft Store. 2 reasons:

1. The Microsoft Store will still be here for at least another year.
2. We're creating a new path for getting Chromium-based PWAs into the existing Store. We'll have guidance on that likely this month.

That said, I'm flexible. If you feel it's better to highlight AppSource here instead of the Store, I'll change it.

Reply:
I just want to use the correct branding. Do you have a link to the current store?

Reply:
I've updated this to clarify that Microsoft Store = the commercial app store shipped with Windows.